R/max_gap.R

```diff
@@ -170,11 +177,18 @@ find_all_meaningful_gap <- function(x, change_points) {
 #' }
 #' @export
 meaningful_gaps_local <- function(x, seg_points, change_points, min_gap = 2) {
+  UseMethod('meaningful_gaps_local')
+}
+
+#' @exportS3Method meaningful_gaps_local Histogram
+meaningful_gaps_local.Histogram <- function(x, seg_points, change_points, min_gap = 2) {
+  meaningful_gaps_local.numeric(x = x$histogram_data, seg_points = seg_points, change_points = change_points, min_gap = min_gap)
+}
+
+#' @exportS3Method meaningful_gaps_local numeric
+meaningful_gaps_local.numeric <- function(x, seg_points, change_points, min_gap = 2) {
```

Comment:
Use `@rdname meaningful_gaps_local` to link all pages together. See other comment with more info below